content/id/docs/concepts/cluster-administration/addons.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Instalasi Addons
```

Comment:
Wondering if we should use Add-ons or Addons

Reply:
I prefer Add-ons, easier to read

Reply:
please fix it then and change it to Add-ons